src/commands/__tests__/install.test.js

```diff
@@ -17,6 +18,8 @@ jest.unmock('../install');
 
 const unsafeProcesses: any & typeof processes = processes;
 const unsafeYarn: any & typeof yarn = yarn;
+const unsafeConstants: any & typeof constants = constants;
```

Comment:
Most likely don't need these, we just use them when you need to mock a function and you need to inspect the .mocks property, but still retain other type information.

Just fyi, no need to change

Reply:
Ah, you did do that for processes too